Documentation/SpatialAwareness/SpatialObjectMeshObserver.md

```diff
@@ -1,79 +1,78 @@
-# Configuring Mesh Observers for Editor
+# The Spatial Object Mesh Observer
```

Comment:
```suggestion
# The spatial object mesh observer
```

Reply:
@thalbern Should headers / titles for the entire page stay in caps?

Reply:
no, they should be lower case - it's ms docs styleguide - don't do random capitalization  - only names should be uppercase